win32screenshot.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |s|
   
   s.add_dependency("ffi", "~> 1.15.0")
   s.add_dependency("mini_magick", "~> 4.9")
-  s.add_dependency("rautomation", "~> 1.1.0")
+  s.add_dependency("rautomation", "~> 2.1")
```

Comment:
I guess it should have been 2.0 in here? There's no 2.1 yet.

Reply:
You're right. Fixed